tests/expectations/test_expectation_arguments.py

```diff
@@ -525,20 +600,24 @@ def test_result_format_configured_no_set_default_override(
         **expectation_arguments_without_meta, **expectation_meta
     )
     result = validator.expect_column_values_to_not_be_null(**expectation_parameters)
-    assert len(result.result.keys()) > 0
+    assert len(result.result.keys()) == 0
 
 
 def test_result_format_configured_with_set_default_override(
     in_memory_runtime_context, test_spark_df
 ):
     catch_exceptions: bool = False  # expect exceptions to be raised
-    result_format: str = "SUMMARY"
-    runtime_environment_arguments = {
+    result_format: str
+
+    result_format = "SUMMARY"
```

Comment:
Please change all instances where `result_format` is string to dictionary.

Reply:
Done (in next commit).